website/templates/project/project.mako

```diff
@@ -16,7 +16,7 @@
                         </h2>
                     % else:
                         <h2 class="node-parent-title unavailable">
-                            <span>Private Project</span>&nbsp;/
+                            <span>Private Project</span> <i class="fa fa-level-down fa fa-dark-lg"> </i>
```

Comment:
I think I prefer the `/` to the bent arrow as a separator. I think it better fits the "projects-as-filesystems" analogy. Thoughts?